lib/arel/nodes/ordering.rb

```diff
@@ -0,0 +1,27 @@
+module Arel
+  module Nodes
+    class Ordering < Unary
+      REVERSE_NULLS = { first: :last, last: :first }
+
+      attr_accessor :nulls
+
+      def initialize(expr, options = {})
```

Comment:
I played around with the alternative, and it reads much better as `nulls: :first`

I do find myself wanting to
`order(column, nulls: :first)` instead of `order(column).asc(nulls: :first)` - but that is minor and wasn't as quick a fix as I first thought